### PR TITLE
DDAU Ember Checkbox List

### PR DESCRIPTION
**Pull request by nicomihalich**
[Rendered](https://github.com/DockYard/reefpoints/blob/982e3bd7721160d5038c65db4d29f8aaea3f7489/source/posts/2016-11-18-checkbox-list-ember.md)

### CODE REVIEW

File: source/posts/2016-11-18-checkbox-list-ember.md [hunk 1]
```diff
@@ -0,0 +1,144 @@
+---
+layout: post
+title: "Making a DDAU checkbox list in Ember.js"
+social: true
+author: Nico Mihalich
+github: nicomihalich
+summary: ""
```
Comment (nicomihalich) [inline source/posts/2016-11-18-checkbox-list-ember.md]: I forgot a summary. Will update

File: source/posts/2016-11-18-checkbox-list-ember.md [hunk 2]
```diff
@@ -0,0 +1,144 @@
...
+Also we'll need some options to select from.  Let's put that in the controller:
+
+```javascript
+import Ember from 'ember';
+
+const { Controller, set } = Ember;
+
+export default Controller.extend({
+  init() {
+    set(this, 'playerOptions', [
```
Comment (brzpegasus) [inline source/posts/2016-11-18-checkbox-list-ember.md]: Always call `this._super(...arguments);` when overriding these lifecycle hooks, especially `init`.

**Comment by brzpegasus** [comment]
Looks good, @nicomihalich!

**Comment by nicomihalich** [comment]
@brzpegasus Updated! Also updated the gist to match the code change

**Pull request merged**